ts/src/index.ts

```diff
@@ -84,11 +84,11 @@ export async function initConfig(config: Config): Promise<ProfilerConfig> {
                 (err: Error) => {
                     // ignore errors, which will occur when not on GCE.
                 }) ||
-        ['', ''];
-    if (!mergedConfig.zone) {
+        [undefined, undefined];
```

Comment:
`[]` would be adequate.

Reply:
I tried that, but got the following error:

ts/src/index.ts(78,12): error TS2525: Initializer provides no value for this binding element and the binding element has no default value.

78     const [instance, zone] =